lib/management.js

```diff
@@ -163,6 +164,23 @@ function patchConfiguration(instanceId, newConf, log, cb) {
                     'constraints', { error });
                 return cb(error);
             }
+            try {
+                const locationsWithNoScality = Object.keys(locations)
```

Comment:
What about naming this `locationsWithReplicationBackend`?

Reply:
Since I changed the way I filter it, your constant naming makes more sense. Thanks for the review @jonathan-gramain